cmd/node/main.go

```diff
@@ -611,6 +611,13 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	}
 	log.Debug("config", "file", ctx.GlobalString(nodesFile.Name))
 
+	if generalConfig.Hardfork.AfterHardFork {
+		log.Debug("changed genesis time after hardfork",
+			"original genesis time", genesisNodesConfig.StartTime,
```

Comment:
"old genesis time", as the new genesis time will become genesisNodesConfig.StartTime so the original genesis time will not exist anymore after the second hard fork.

Reply:
done